.openshift-ci/post_tests.py

```diff
@@ -204,6 +209,16 @@ def check_stackrox_logs(self):
             timeout=PostTestsConstants.CHECK_TIMEOUT,
         )
 
+    def add_test_result(self, test_results):
+        print("Storing operator test results in JUnit format")
+        for fromDir, toDir in test_results:
+            self.run_with_best_effort(
+                ["scripts/ci/store-artifacts.sh", "store_test_results",
+                 fromDir, toDir],
```

Comment:
The rest of this file seems to avoid camelCase.
```suggestion
        for from_dir, to_dir in test_results:
            self.run_with_best_effort(
                ["scripts/ci/store-artifacts.sh", "store_test_results",
                 from_dir, to_dir],
```